render-tests/generate_pmtiles.sh

```diff
@@ -0,0 +1,9 @@
+mkdir -p pmtiles
+cd ../tiles
+mvn clean package -DskipTests
+java -Xmx2g -jar target/protomaps-basemap-HEAD-with-deps.jar \
+    --osm_path=../render-tests/fixtures/monaco-2025-02-15.osm.pbf \
```

Comment:
I think in the future we can use SliceOSM to generate a more linguistically diverse, yet compact OSM extract and then render the PMTiles off of that